README.md

```diff
@@ -65,14 +65,19 @@ install all the dependencies.
 
 The application uses Postgres for the database, so you need to have that on
 your server. You need to create a database, which you can do with `createdb
-gavel` (unless you're using a different database name). Before you use the app,
-you need to initialize the database by running `python initialize.py`. **Note
-that Gavel does not preserve database schema compatibility between versions.**
+gavel` (unless you're using a different database name). In order to send 
+emails, you'll need a broker on the server. You can do this using redis by
+running `redis-server`. Before you use the app,you need to initialize the 
```

Comment:
Just like we don't tell people how to install and run Postgres, I think we shouldn't explicitly tell people how to run Redis -- we should just tell them that they need to have Redis running. (Most likely, people won't be running it manually via `redis-server` -- they'll be using init scripts or something similar to run Redis as a service.)